agasc/agasc.py

```diff
@@ -397,29 +509,67 @@ def get_stars(ids, agasc_file=None, dates=None, fix_color1=True, use_mag_est=Fal
         update_color1_column(t)
     t['DATE'] = dates
 
-    if use_mag_est:
-        update_mags_from_supplement(t)
+    if use_supplement:
+        update_from_supplement(t)
 
     return t if ids.shape else t[0]
 
 
-def update_mags_from_supplement(stars):
-    """Overwrite mag and color1 information from AGASC supplement in ``stars``.
+# Interpolate COMMON_DOC into those function docstrings
+for func in get_stars, get_star, get_agasc_cone:
+    func.__doc__ = func.__doc__.format(common_doc=COMMON_DOC)
+
+
+def update_from_supplement(stars):
```

Comment:
And this is another that could be in the supplement namespace. I.e. `agasc.supplement.update_star` instead of `agasc.update_from_supplement`

Reply:
This is really an internal helper function for `get_stars` and `get_cone_agasc`. It is tightly integrated with those functions and not exported from the module. So I'd rather keep it here in the module that is providing user access to the AGASC.